packages/kotti-ui/source/kotti-field-number/KtFieldNumber.vue

```diff
@@ -1,18 +1,23 @@
 <template>
 	<KtField v-bind="{ field }" :helpTextSlot="$slots.helpText">
 		<div
+			ref="wrapperRef"
 			class="kt-field-number"
 			:class="{
 				'kt-field-number--is-hide-change-buttons': hideChangeButtons,
 				'kt-field-number--has-maximum': showMaximum && maximum !== null,
 			}"
+			:tabIndex="0"
```

Comment:
Is this `tabIndex` needed? 
In FT you need to tab twice to get to the `decrement` button and be able to triggered with the `Enter` key.

Reply:
i removed this tabIndex. When testing with a `KtfieldNumber` not in a first position: after firstname for example. Then we just need to tab once to get in the KtFieldNumber

Reply:
@santiagoballadares I requested that he brings this back because it means that the div captures focus in the first place
and so if we remove this, we should also remove the checks from the isEventTarget